pkg/apis/crds/v1alpha1/workerroute_types.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Would be nice to maintain [the license](https://github.com/hellofresh/kubeflare/blob/master/hack/boilerplate.go.txt) used in the rest of the codebase.
I think `make generate` should use the expected license